vulnerabilities/migrations/0002_vulnerabilityseverityscore.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.0.7 on 2020-11-27 13:43
```

Comment:
Could some of these migrations be squashed in a fewer number of migrations, for clarity?
We are creating then deleting then creating something else... all these details do not make sense to keep

Reply:
Yes, I  will squash this once everything else is fine.